sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -0,0 +1,271 @@
+# Azure Communication Phone Numbers client library for Java
+
+The phone numbers package provides capabilities for Phone Number Management.
+
+Acquired phone numbers can come with many capabilities, depending on the country, number type and phone plan. Examples of capabilities are SMS inbound and outbound usage, PSTN inbound and outbound usage. Phone numbers can also be assigned to a bot via a webhook URL.
+
+[Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][product_docs]
+## Getting started
+
+### Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) version 8 or above.
+- [Apache Maven](https://maven.apache.org/download.cgi).
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-communication-phonenumbers;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-phonenumbers</artifactId>
+  <version>1.0.0-beta.4</version>
+</dependency>
+```
+
+## Key concepts
+
+
+### Initializing Phone Number Client
+The PhoneNumberClientBuilder is enabled to use Azure Active Directory Authentication
```

Comment:
Looking forward to have this updated with the new `PhoneNumbersClient` API soon and then compare with before :-)